### PR TITLE
add babel plugin for displayName to README

### PR DESCRIPTION
**Pull request by bernard-lin**
**What**: Adding a link for the babel plugin for displayNames to the README and amending the description.

**Why**: Addresses issue #29 

**How**: By editing the README and adding a link to the npm package.

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Comment (kentcdodds) [inline npm-debug.log]: Could you remove this?

Comment (kentcdodds) [inline npm-debug.log]: Maybe you could add this to the `.gitignore`? :)

Comment (bernard-lin) [inline npm-debug.log]: Haha oops!

**Comment by bernard-lin** [comment]
npm-🚫🐜. 🌲 removed!

**Comment by bernard-lin** [comment]
Yay! Thanks so much for your help and guidance - the YouTube/PayPal/Twitch talks, FEM, everything. This is my first time making a meaningful contribution to an open source project. Can't wait to see where it goes from here 😄

**Comment by kentcdodds** [comment]
That's awesome! Good work!!

**Pull request merged**